src/modules/dashboard/components/Dialogs/ControlSafeDialog/SafeTransactionPreview.tsx

```diff
@@ -16,9 +17,10 @@ import Avatar from '~core/Avatar';
 import MaskedAddress from '~core/MaskedAddress';
 import { AnyUser, Colony } from '~data/index';
 import { AbiItemExtended, getArrayFromString } from '~utils/safes';
+import { omit } from '~utils/lodash';
```

Comment:
Is there a difference between this `omit` and the one I commented on earlier?

Reply:
This one is the pattern we should follow, and any additional imports added to this file and imported from there. Keeps bundle size down and allows to group lodash imports on a single line